### ISSUE TITLE
Unification of line feed codes

### ISSUE THREAD

**Issue by btyanagawamg** [terasolunaorg/terasoluna-gfw#763]
## Description
Now, line feed code is mixed LF and CRLF.
We unify line feed codes in this repository to LF.

## Possible Solutions
- Unify line feed codes in this repository to LF.

## Affects Version/s
- 5.X.X.RELEASE
- 1.X.X.RELEASE

## Fix Version/s
- [x] 5.4.0 (master)
- [x] 5.3.1 (5.3.x)
- [x] 5.2.2 (5.2.x)
- [x] 5.1.2 (5.1.x)
- [x] 5.0.3 (5.0.x)
- [x] 1.0.7 (1.0.x)

## Final Solution
- Unify line feed codes in this repository to LF.

## Migrating Guide
- Not Required

## Issue Links
- #XXX

**Comment by yoshikawaa** [comment]
@btyanagawamg 
I think it's better to adjust encoding setting between `formatter-maven-plugin` and `maven-license-plugin`.

* formatter use default `${project.build.sourceEncoding}` (UTF-8).
* license use custom `${encoding}` (UTF-8).

How do you think?

**Comment by btyanagawamg** [comment]
@yoshikawaa 
Yes, I think that it is better to unify the settings that define character codes .

**Comment by yoshikawaa** [comment]
@btyanagawamg 
OK, I reopen this. But we can do it when have time.

**Comment by bttakahashikkr** [comment]
@yoshikawaa @btyanagawamg
The encoding property may not be necessary , because many plugins refer to the project encoding property as default values.
However, this issue is put on hold for the following reasons.
 - It takes a lot of time to check the POM setting and delete the encoding property.
 - It takes a lot of time to check whether the encoding setting is necessary, and further check whether the project encoding property is referenced as the default value , for all plugins.
